sdk/monitor/monitor-query/test/internal/unit/logsQueryClient.unittest.spec.ts

```diff
@@ -30,11 +30,11 @@ describe("LogsQueryClient unit tests", () => {
 
     const client = new LogsQueryClient(tokenCredential, {
       endpoint: "https://customEndpoint1",
-      scopes: "customScopes1"
+      scopes: "https://customscopes1/"
     });
 
     assert.equal(client["_logAnalytics"].$host, "https://customEndpoint1");
-    assert.equal(client["_logAnalytics"]["baseUri"], "https://customEndpoint1");
+    assert.equal(client["_logAnalytics"]["_baseUri"], "https://customEndpoint1");
```

Comment:
wow, you're accessing private class fields for fun here! a true hacker!

Reply:
Hahaha, i wasn't supposed to do that :P

Reply:
I mean it is fine, at least for testing, just make sure that pattern does not show up in customer-facing samples 😄